src/panels/developer-tools/action/developer-tools-action.ts

```diff
@@ -478,7 +486,49 @@ class HaPanelDevAction extends LitElement {
     }
     button.progress = true;
     try {
-      this._response = (await callExecuteScript(this.hass, script)).response;
+      const result = (await callExecuteScript(this.hass, script)).response;
+      this._response = {
+        domain,
+        service,
+        result,
+        media:
+          "media_source_id" in result
+            ? resolveMediaSource(this.hass, result.media_source_id).then(
+                (resolved) =>
+                  resolved.mime_type.startsWith("image/")
+                    ? html`<img src=${resolved.url} alt="Media content" />`
+                    : resolved.mime_type.startsWith("video/")
+                      ? html`
+                          <video
+                            controls
+                            src=${resolved.url}
+                            alt="Video content"
+                          ></video>
+                        `
+                      : resolved.mime_type.startsWith("audio/")
+                        ? html`
+                            <audio
+                              controls
+                              src=${resolved.url}
+                              alt="Audio content"
+                            ></audio>
+                          `
+                        : html`
+                            <a
+                              href=${resolved.url}
+                              target="_blank"
+                              rel="noreferrer"
+                              ><ha-button>
+                                ${this.hass.localize(
+                                  "ui.panel.developer-tools.tabs.actions.open_media",
+                                  { media: result.media_source_id }
```

Comment:
You pass a variable to the translation, but it is unused in the translation string